index.html

```diff
@@ -3,9 +3,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Welcome to potionfy</title>
 </head>
 <body>
-    <p>Welcome to potionfy</p>
+    <p>Welcome to potionfy! Enjoy your stay!!</p>
```

Comment:
Could you remove 'Enjoy your stay!' string and make the string p tag to red?

Reply:
Sure! Can do